File: TH_Buoi2/ViDu2_Bai2.jl
function linear_search(arr, target)
    for (index, value) in enumerate(arr)
        if value == target
            return index
        end
    end
    return -1
end

arr = [4, 2, 5, 1, 3]
println(linear_search(arr, 5)) # Output: 2

File: TH_Buoi2/ViDu2_Bai2.cpp
#include <iostream>
#include <vector>

int linear_search(const std::vector<int>& arr, int target) {
    for (size_t index = 0; index < arr.size(); ++index) {
        if (arr[index] == target) {
            return index;
        }
    }
    return -1;
}

int main() {
    std::vector<int> arr = {4, 2, 5, 1, 3};
    std::cout << linear_search(arr, 5) << std::endl; // Output: 2
    return 0;
}